packages/tradershub/src/components/ActionScreen/ActionScreen.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { ComponentProps, isValidElement, ReactNode } from 'react';
-import { qtMerge } from '@deriv/quill-design';
+import { clsx } from 'clsx';
```

Comment:
I suggest utilizing `twMerge` rather than `clsx`. It is made for tailwind which has similar functionalities to `qtMerge`

Reply:
Nice, but twMerge is quite the bundle size compared to clsx

Reply:
Yeah it might be heavier on the bundle size however the functionality is really useful especially for projects that are fully utilizing tailwind. 

Another suggestion is to create a function where you can use both tailwind-merge and clsx hand in hand. So you can have best of the both worlds.

Reply:
The versioning is different compared to quill-design, so its better if deriv-com/ui team can give us the utility function with the latest version 😸

Reply:
I agree with @aizad-deriv . we can use the following piece of code for this matter:

```
import { ClassValue, clsx } from 'clsx'
import { twMerge } from 'tailwind-merge'

export const cn = (...inputs: ClassValue[]) => {
  return twMerge(clsx(inputs))
}
```

Reply:
yes @hirad-deriv, @aizad-deriv will give us the utility function on his end